README.md

```diff
@@ -35,31 +37,77 @@ have different degree of access throughout the application.
 
 ---
 
-### Deployment
+## 3. Deployment
+
+### Alternative 1: Download/Clone Repository
+
+- Download the [Latest Release](https://github.com/Patlenlix/CrimeDatabase/releases)
+    - Unzip the compressed file where you want to save the application
 
-1. Clone/Fork this repo in your favorite IDE
-    - Go to the folder where you want the application to save 
+**OR**
+
+- Clone the repository
```

Comment:
This will give them the entire repo (not only the released version). How do we solve this? Is it "enoghf" to open the code in intelliJ and checkout the commit with the wanted tag (the latest release)? or will it run the main-branch? not sure...

Reply:
The Deployment instructions doesn't really have anything to do with the release of the project. These are just different ways to access the app. 

But I see your point, might be nice to just access the latest release and not the current version of the main we are merging into. I guess this is why companies usually have different branches for release and deployment.

I think there might be a way to use Git and clone the repo from a specific tag/release?

Reply:
Yes, I guess thats true... 

We can let this be as it is for now :)

Reply:
I think so, and we could make an issue to look into this later.